src/Get-Help/User-Guide/Installation/index.md

```diff
@@ -67,6 +67,105 @@ The ZIP can be installed on all Linux distros. It includes the desktop
 application, the server and the command line tool. To install, simply
 extract the contents to any destination you want.
 
+### Docker
+
+The Docker distribution is not available as a download on the
+website. It comes in the form of a Docker image that you can obtain
+via the Docker command line interface:
+
+~~~sh
+docker pull daisyorg/pipeline-assembly
+~~~
+
+It is also possible to get a specific version:
+
+~~~sh
+docker pull daisyorg/pipeline-assembly:<version>
+~~~
+
+You can find the available versions at
+[https://hub.docker.com/r/daisyorg/pipeline-assembly/tags](https://hub.docker.com/r/daisyorg/pipeline-assembly/tags). After
+having pulled the image you're ready to run the Pipeline web server:
+
+~~~sh
+docker run --detach \
+           -p 8181:8181 \
+           daisyorg/pipeline-assembly
+~~~
+
+This will make the web service available on the address
+http://localhost:8181/ws. By default, authentication is enabled. The
+key and secret are "clientid" and
+"sekret". [Environment variables]({{site.baseurl}}/Get-Help/User-Guide/Pipeline-as-Service/#environment-variables)
+can be provided with one or more `-e` arguments.
```

Comment:
I would add an example with the `-e` arguments

Reply:
I had an example first but then removed it because there are examples below.